_config.yml

```diff
@@ -25,6 +25,9 @@ imageNames:
   kubeControllers: quay.io/calico/kube-controllers
   calico-upgrade: quay.io/calico/upgrade
   flannel: quay.io/coreos/flannel
+  dikastes: quay.io/calico/dikastes
+  pilot-webhook: quay.io/calico/pilot-webhook
+  flexvol: quay.io/saurabh/flexvol
```

Comment:
Will this image be available under `saurabh` or do we plan to move it into `calico`?

Reply:
Saurabh is up-streaming it to Istio